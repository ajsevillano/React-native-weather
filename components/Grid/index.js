import { Text, View } from 'react-native';

const Grid = () => {
  return (
    <View>
      <Text>Grid</Text>
      <Text>Grid</Text>
    </View>
  );
};

export default Grid;
