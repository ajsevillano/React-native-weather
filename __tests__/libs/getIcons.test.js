// import AsyncStorage from '@react-native-async-storage/async-storage';

// const checkFirstTimeUsingApp = async () => {
//   // Getting the firstTimeRunningApp of the key `isFirstTime` from the AsyncStorage.
//   await AsyncStorage.getItem('isFirstTime');
// };

// it('checks if Async Storage is used', async () => {
//   await checkFirstTimeUsingApp();

//   expect(AsyncStorage.getItem).toBeCalledWith('isFirstTime');
// });

import React from 'react';
import renderer from 'react-test-renderer';

import Grid from '../../components/Grid';

describe('<Grid />', () => {
  it('has 1 child', () => {
    const tree = renderer.create(<Grid />).toJSON();
    console.log(tree.length);
    expect(tree).toMatchSnapshot();
  });
});
