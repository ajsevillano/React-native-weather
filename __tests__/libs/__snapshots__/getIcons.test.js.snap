// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Grid /> has 1 child 1`] = `
<View>
  <Text>
    Grid
  </Text>
  <Text>
    Grid
  </Text>
</View>
`;
